packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -17,14 +17,10 @@ export const BlockLoaderInput: React.VFC = () => {
   const [blockUrl, setBlockUrl] = useState("");
   const blockUrlRef = useRef<HTMLInputElement | null>(null);
 
-  const isDefinedBlock = userBlocks.some(
-    (userBlock) =>
-      createNormalizedBlockUrl(userBlock.componentId) ===
-      createNormalizedBlockUrl(blockUrl),
-  );
+  const isDefinedBlock = !!userBlocks[createNormalizedBlockUrl(blockUrl)];
   const isValidBlockUrl = Boolean(blockUrlRef.current?.validity.valid);
 
-  const inputDisabled = isDefinedBlock || !isValidBlockUrl || error != null;
+  const inputDisabled = !isValidBlockUrl || error != null;
```

Comment:
Allowing blocks to be reloaded if the user chooses, to override blocks that have changed.